addons/mail/static/src/web/messaging_menu/messaging_menu.xml

```diff
@@ -77,7 +77,7 @@
             <t t-foreach="displayedPreviews" t-as="preview" t-key="preview.id">
                 <NotificationItem
                     body="preview.message?.inlineBody"
-                    count="preview.count"
+                    counter="preview.isNeedaction ? preview.counter : preview.counter > 1 ? preview.counter : undefined"
```

Comment:
counter for DM/group should be unread counter, not needaction

Reply:
Done! I have also added a test for chat unread counter in messaging menu